languoids/glottocodes.json

```diff
@@ -9140,6 +9140,7 @@
     "unal": 1234,
     "unam": 1242,
     "unat": 1236,
+    "uncl": 1508,
```

Comment:
Hm. That looks like some weird shit going on here. There must never be duplicate keys in this dictionary - and I guess that happened when you merged some unspecified state of master into this branch.